core/mm3Types.js

```diff
@@ -16,11 +16,109 @@
  */
 
 CLASS({
+  name: 'Message',
```

Comment:
I don't think we should put this in mm3Types.  Maybe add mm2.5Message or something?